project_template/webpack.config.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
   output: {
     path: out,
     filename: "[name].[hash].js",
-    publicPath: "./"
+    publicPath: "/"
```

Comment:
Was this change needed for nightwatch? This likely breaks in Cordova, because we need relative file based urls for Cordova.